### DISCUSSION TITLE
Role Icons

### DISCUSSION THREAD

**Discussion by EthanWinters2885** [stoatchat/discussions#688] [Feature Suggestions]
### What do you want to see?

It would be pretty epic if there was role icons on Revolt, just like how Discord & Guilded have them, and with the ability to add static or animated role icons

**Comment by EthanWinters2885** [comment]
I hope this will get some attention, I wanna also know what other people think too

**Reply by AdamTheEmperor** [reply]
we think the same men it will be the best feature for now

**Reply by Taureon** [reply]
> we think the same men it will be the best feature for now

what does that mean

**Reply by Taureon** [reply]
> I hope this will get some attention, I wanna also know what other people think too

yeah i like it! i wonder if there'll be more options to how emojis are added to the roles?
i feel like there's more ways to be creative here

**Comment by Dadadah** [comment]
Role Icons were [added recently](https://github.com/stoatchat/stoatchat/commit/841985d3b994df1c6eefab2fc7ecbd77ab22c493) to the backend, and will be shipped in v0.13.0. There is a [git issue](https://github.com/stoatchat/for-web/issues/1156) to implement it in the for-web repo.
